pkg/api/types.go

```diff
@@ -62,6 +62,9 @@ type Config struct {
 	// variables.
 	EnvironmentFile string
 
+	// BuildNamespace provides the namespace under which the labels will be generated.
+	BuildNamespaceLabel string
```

Comment:
`LabelNamespace`, and moreover make the comment (one line above) match the name.